docs/faq.md

```diff
@@ -83,8 +83,8 @@ If your package will be used as an application, it might be worth to define an u
 
 ### Is tox supported?
 
-**Yes**. By using the [isolated builds](https://tox.readthedocs.io/en/latest/config.html#conf-isolated_build) `tox` provides,
-you can use it in combination with the PEP 517 compliant build system provided by Poetry.
+**Yes**. Provided that you are using `tox` > 4, you can use it in combination with the PEP 517 compliant build system
```

Comment:
Shouldn't it be `tox >= 4`?

We could also add a sentence that you have to set `isolated_build` for tox 3 (with an updated link to https://tox.wiki/en/3.27.1/config.html#conf-isolated_build). No strong feelings about that.

Reply:
OK I will implement both feedbacks. Thank you.